package/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint global-require: 0 */
+/* eslint import/no-dynamic-require: 0 */
+
+const Environment = require('./environment')
+const { existsSync } = require('fs')
+
+function createEnvironment() {
+  const path = `./environments/${process.env.NODE_ENV}`
```

Comment:
@javan Missing file extension here

Reply:
Not needed. I removed them all throughout the codebase. Just like Ruby, Node's `require` doesn't need an extension.

Reply:
but `existSync` won't work, it requires a path 

```
const path = resolve(__dirname, `./environments/${process.env.NODE_ENV}.js`)
```

Reply:
```  
const constructor = existsSync(path) ? require(path) : Environment
```

Will always evaluate to false and return default config

Reply:
D'oh, right you are. Great catch! Fixed in df21faead2b427c56152eca744f12ce8fcc1e0ca